content/blog/new-kubernetes-superpowers/index.md

```diff
@@ -0,0 +1,333 @@
+        _, err := yaml.NewConfigFile(ctx, "my-certificate-definition",
+			&yaml.ConfigFileArgs{
+				File: "certificate.yaml",
+			},
+		)
+		if err != nil {
+			return err
+		}
+
+		// Instantiate a Certificate resource.
+        _, err = certv1b1.NewCertificate(ctx, "example-cert", &certv1b1.CertificateArgs{
+			Metadata: &metav1.ObjectMetaArgs{
+				Name: pulumi.String("example-com"),
+			},
+			Spec: certv1b1.CertificateSpecArgs{
+				SecretName:  pulumi.String("example-com-tls"),
+				Duration:    pulumi.String("2160h"),
+				RenewBefore: pulumi.String("360h"),
+				CommonName:  pulumi.String("example.com"),
+				DnsNames: pulumi.StringArray{
+					pulumi.String("example.com"),
+					pulumi.String("www.example.com"),
+				},
+				IssuerRef: certv1b1.CertificateSpecIssuerRefArgs{
+					Name: pulumi.String("ca-issuer"),
+					Kind: pulumi.String("Issuer"),
+				},
+			},
+		})
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+## OPA support in Pulumi CrossGuard Policy-as-Code
+
+Pulumi’s [CrossGuard]({{< relref "/docs/guides/crossguard" >}}) policy-as-code framework enforces policy across all your modern cloud infrastructure and/or in Kubernetes - using the full expressiveness of rich programming languages. It is deeply integrated into the deployment process to ensure that policy violations are identified *before* infrastructure is deployed instead of minutes or hours after changes are made.
+
+Today’s release of [Open Policy Agent](https://www.openpolicyagent.org/) (OPA) support in Pulumi policy-as-code extends the existing support for TypeScript and Python to include OPA’s `.rego` language and runtime for declarative policy specification. As an emerging standard for cloud native policy, OPA is already used across a wide variety of cloud native technologies, and can now also be leveraged to author policies to run in Pulumi’s policy model across any cloud or Kubernetes.
+
+OPA-based rules can be authored using Pulumi’s rich resource model (top pane) and enforced as part of deployments to any infrastructure (bottom pane).
+
+![Pulumi OPA CrossGuard VSCode](opa-crossguard-vscode.png)
+
+And as part of Pulumi’s support for organizational enforcement of policy, OPA-based policy can be enforced across all infrastructure deployments in your organization (including across different clouds and in Kubernetes clusters).
+
+![Pulumi OPA CrossGuard Console](opa-crossguard-console.png)
+
+## Helm 3 and Kustomize
+
+Pulumi's Kubernetes support has long included support for both defining Kubernetes resources directly in your Pulumi program, as well as using Pulumi to deploy Kubernetes resources define in existing YAML or Helm. With a wealth of existing YAML and Helm content out in the ecosystem, these features have been used heavily by cloud engineering teams to deploy critical Kubernetes platform services and application logic into their Kuberentes clusters.  
```

Comment:
```suggestion
Pulumi's Kubernetes support has long included support for both defining Kubernetes resources directly in your Pulumi program, as well as using Pulumi to deploy Kubernetes resources from existing YAML or Helm. With a wealth of existing YAML and Helm content out in the ecosystem, these features have been used heavily by cloud engineering teams to deploy critical Kubernetes platform services and application logic into their Kubernetes clusters.  
```

Reply:
Thanks - not sure why your previous fix for this didn't get pulled in here!